support.md

```diff
@@ -49,23 +49,64 @@ reported issues within 2 business days.
 
 ## Releases
 
-We regularly publish [binary releases of Bazel](https://github.com/bazelbuild/bazel/releases). To
-that end, we announce release candidates on
-[bazel-discuss](https://groups.google.com/forum/#!forum/bazel-discuss); these are binaries that have
-passed all of our unit tests. Over the next few days, we regression test all applicable build
-targets at Google. If you have a critical project using Bazel, we recommend that you establish an
-automated testing process that tracks the current release candidate, and report any regressions.
-
-If no regressions are discovered, we officially release the candidate after a week. However,
+We regularly publish [binary releases of Bazel](https://github.com/bazelbuild/bazel/releases).
+
+### Policy
+
+Every beginning of the month (we target the first business day of the month), we create a new release
+candidate for a new MINOR version (e.g. 0.6.0). The work is tracked by a [release bug on GitHub](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+is%3Aopen+label%3Arelease) which indicates
+the exact target date for the incoming month and assigned to the current Release Manager.
+Those release candidate should pass all our unit tests, and show no unwanted regression in the
+projects tested on [ci.bazel.io](http://ci.bazel.io).
+
+We announce those release candidates on [bazel-discuss](https://groups.google.com/forum/#!forum/bazel-discuss);
+these are binaries that have passed all of our unit tests. Over the next days, we monitor community bug
+reports for regressions in release candidate.
+
+If no regressions are discovered, we officially release the candidate after two weeks. However,
 regressions can delay the release of a release candidate. If regressions are found, we apply
 corresponding cherry-picks to the release candidate to fix those regressions. If no further
-regressions are found for two business days, but not before a week has elapsed since the first
+regressions are found for two business days, but not before two week has elapsed since the first
 release candidate, we release it.
 
+After the release candidate is cut, we do not cherry-pick new features into release candidates.
```

Comment:
style: "we do not cherry-pick ... into **them**" (avoid repetition)

Reply:
I used 'After *a* release ..., into *it*'